src/app/toolbar/toolbar.component.html

```diff
@@ -15,85 +15,101 @@
 -->
 
 <mat-toolbar class="toolbar">
-  <mat-toolbar-row class="toolbar-row">
-    <a routerLink="/home" class="title-wrapper">
-      <div class="icon-placeholder"></div>
-      <span class="title">SafeNet</span>
-    </a>
-    <ng-container *ngIf="userIsSignedIn">
-      <!-- Note: This is how the spec for aria-current specifies it should be used, but this does not seem to be read by the Chrome screenreader at this point in time. This should be revisited later.-->
-      <div class="navigation-links" aria-label="navigation">
+  <div class="logo-toolbar-wrapper">
+    <mat-toolbar-row class="toolbar-row">
```

Comment:
Why do we need these wrappers? should just be `toolbar` and then multiple `toolbar-row`.